addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2509,6 +2509,7 @@ options.registry.DeviceVisibility = options.Class.extend({
             },
         });
         this.trigger_up('snippet_option_visibility_update', {show: widgetValue !== (isMobile ? 'no_mobile' : 'no_desktop')});
+        this.$target.trigger('content_changed');
```

Comment:
Why is that necessary? It is normally automatic at the end of all snippet option method calls?
```javascript
        // We trigger the event on elements targeted by apply-to if any as
        // this.$target could not be in an editable element while the elements
        // targeted by apply-to are.
        ($applyTo || this.$target).trigger('content_changed');
```